master/getting-started/mesos/installation/dc-os/framework.md

```diff
@@ -1,41 +1,19 @@
 ---
-title: Framework Install Walkthrough
+title: Calico-DC/OS Install Walkthrough
 ---
 
-Calico maintains a Framework for DC/OS for simple installation and use.
-The framework is flexible so that users can use it as an installer,
-a runtime daemon, or both. Below, we'll see exactly how to deploy the following
-common configurations:
+Calico maintains a Mesos Framework which provides a means for simple installation
```

Comment:
It's subtle, but we should stop wording things like "Calico maintains blah" in favor of wording like "Calico can be installed blah"